docs/src/usage.md

```diff
@@ -84,6 +84,7 @@ When Kani builds your code, it does two important things:
 
 1. It sets `cfg(kani)`.
 2. It injects the `kani` crate.
+3. For host crates (crates that your target crate depends on), Kani sets `cfg(kani_host)`.
```

Comment:
1. Line 83 says "two important things". We should increment that. :)
2. I would change the language of 1 and 3.
> 1. It sets `kani` configuration for target crates compilation (including dependencies).
> 3. It sets `kani_host` configuration for host build targets such as build script and procedural macro crates.